client/packages/common/src/ui/layout/tables/material-react-table/useTableDisplayOptions.tsx

```diff
@@ -211,5 +241,16 @@ export const useTableDisplayOptions = <T extends MRT_RowData>(
     muiToolbarAlertBannerProps: {
       sx: { backgroundColor: 'unset' },
     },
+    displayColumnDefOptions: {
+      'mrt-row-select': {
+        size: 50,
+        muiTableHeadCellProps: {
+          align: 'center',
+        },
+        muiTableBodyCellProps: {
+          align: 'center',
+        },
+      },
+    },
```

Comment:
Makes the select checkbox column narrower with icons centered. I didn't bother varying it depending on density state as it didn't seem necessary.